codegen-sbt/src/main/scala/caliban/codegen/CodegenPlugin.scala

```diff
@@ -59,7 +59,7 @@ object CodegenPlugin extends AutoPlugin {
   def execGenCommand(
     helpMsg: String,
     args: List[String],
-    writer: (Document, String, Option[String]) => String
+    writer: (Document, String, Option[String], String) => String
```

Comment:
Can you also add the new option in the Help message a few lines above this? (only for `calibanGenSchema`)